### PR TITLE
TELCODOCS-749: Release Notes for 4.11: workload hints

### PR DESCRIPTION
**Pull request by tmulquee**
Version(s): PR applies only to 4.11.

Issue: https://issues.redhat.com/browse/TELCODOCS-749

Link to doc previews:

Workload hints
[TELCODOCS-416](https://issues.redhat.com//browse/TELCODOCS-416):  [CNF-3205](https://issues.redhat.com//browse/CNF-3205) Telco friendly workload specific PerformanceProfile API (http://file.emea.redhat.com/tmulquee/TELCODOCS-749/release_notes/ocp-4-11-release-notes.html#ocp-4-11-workload-hints-for-nto)

### CODE REVIEW

**Review by marioferh — COMMENTED** [review]
/LGTM for TELCODOCS-416

File: release_notes/ocp-4-11-release-notes.adoc [hunk 1]
```diff
@@ -545,6 +553,9 @@ For more information, see xref:../operators/understanding/olm-packaging-format.h
 ==== Java-based Operators (Technology Preview)
 
 Starting in {product-title} {product-version} as a Technology Preview feature, the Operator SDK includes the tools and libraries to develop a Java-based Operator. Operator developers can take advantage of Java programming language support in the Operator SDK to build a Java-based Operator and manage its lifecycle.
+In earlier versions of {product-title}, the Performance Addon Operator provided automatic, low latency performance tuning for applications. In {product-title} 4.11, these functions are part of the Node Tuning Operator. The Node Tuning Operator is part of the standard installation for {product-title} 4.11. If you upgrade to {product-title} 4.11, the Node Tuning Operator removes the Performance Addon Operator and all related artifacts on startup.
```
Comment (MarSik) [inline release_notes/ocp-4-11-release-notes.adoc:556]: Is this in the right place? Seems to be in between Java based operators and the link to those.

**Comment by MarSik** [comment]
I am fine with the content, but the structure seems wrong

File: release_notes/ocp-4-11-release-notes.adoc (hunk 1 above)
Comment (tmulquee) [reply release_notes/ocp-4-11-release-notes.adoc:556]: I will investigate. It's not my content - at first glance looks like something that happened during merge conflict resolution.

Comment (tmulquee) [reply release_notes/ocp-4-11-release-notes.adoc:620]: Resolved the restructuring issue: merge conflict.

**Review by MarSik — APPROVED** [review]
/lgtm

**Pull request merged**